### ISSUE TITLE
Sourcegraph support

### ISSUE THREAD

**Issue by ignatov** [go-lang-plugin-org/go-lang-idea-plugin#2592]
Will be cool to have an external documentation fetched from Sourcegraph.
See https://sourcegraph.com/github.com/golang/go/-/info/GoPackage/net/http/-/NewRequest
and all requests to API in DevMode.

**Comment by dlsniper** [comment]
I was thinking of this and if we could do this with a side panel for the editor like the Markdown plugin does and render a Web page. Would that work?

**Comment by ignatov** [comment]
It's better to show a small popup with documentation or just open the browser. A side panel is a half backed solution.

**Comment by ignatov** [comment]
Guys from Sourcegraph already have a beta of native application for Mac OS X and a simple plugin for IntelliJ.
Another options – just show usage examples right inside the IntelliJ.

**Comment by dlsniper** [comment]
Right, I'm also part of that Beta. I wonder if it's worth to keep this open then, I think it could be closed.
